osl/interface_linux.go

```diff
@@ -241,8 +241,9 @@ func (n *networkNamespace) AddInterface(srcName, dstPrefix string, options ...If
 	if n.isDefault {
 		i.dstName = i.srcName
 	} else {
-		i.dstName = fmt.Sprintf("%s%d", i.dstName, n.nextIfIndex)
-		n.nextIfIndex++
+		dstNamePrefix := i.dstName
```

Comment:
_dstPrefix_ is the prefix of the interface in the host system - I am trying to modify the one in the container. If I override this it will change the logic in the code following. I do not think this is correct - please clarify.

The only reason I assign _dstName_ to a new _dstNamePrefix_ is that the original data of _dstName_ is to be modified and _dstNamePrefix_ is needed for the increment.